template/template.yaml

```diff
@@ -47,6 +47,11 @@ options:
     requires:
       - unstable-hal
 
+  - !Option
+    name: stack-smashing-protection
+    display_name: Enable stack smashing protection.
+    help: Requires nightly Rust.
```

Comment:
I think we should add the `chips` field as is only available for riscv devices

Reply:
It is?

Reply:
no it's available for Xtensa, too

Reply:
Upps, in my head it was only available for riscv sorry!